crates/rnote-ui/src/dialogs/export.rs

```diff
@@ -421,7 +451,7 @@ pub(crate) async fn dialog_export_doc_pages_w_prefs(appwindow: &RnAppWindow, can
                         }
                     }
                     Err(e) => {
-                        tracing::debug!("Did not export document pages (Error or dialog dismissed by user), Err: {e:?}");
+                        tracing::debug!("mac suffix consistency VERSION Did not export document pages (Error or dialog dismissed by user), Err: {e:?}");
```

Comment:
What's with this message?

Reply:
Oh sorry, that's me changing debug message to be sure that I actually test the right version. I've had issues before with the install not working or the `rnote` command not launching the version I just compiled after compiling it and installing. It was not really meant to be pushed, I removed it

Reply:
ok, no problem!